utils/utils.py

```diff
@@ -94,28 +93,15 @@ def get_first_name(full_name: str) -> str:
     return full_name.split(' ')[0]
 
 
-def render_dialog(text: str, character: str, background: str = 'camp') -> BytesIO:
+def render_dialog(text: str, charactername: str, background: str = 'camp') -> BytesIO:
     background = Image.open('./assets/images/dialog/backgrounds/' + background + '.png')
-    ribbon = Image.open('./assets/images/dialog/ribbons/' + character + '.png')
-    character = Image.open('./assets/images/dialog/characters/' + character + '.png')
+    ribbon = Image.open('./assets/images/dialog/ribbons/' + charactername + '.png')
```

Comment:
You should use the character aliases we have for the quote command, they can be moved to this file and imported where needed